### PR TITLE
rm: terra-gnome-shell-extension-pop-shell

### PR DESCRIPTION
**Pull request by sadlerm4**
Obsoleting because the package exists in Fedora.

Thankfully the 1st release of the latest version of this package is uninstallable on 41 anyway.

### CODE REVIEW

File: anda/terra/obsolete/terra-obsolete.spec [hunk 1]
```diff
@@ -121,6 +121,8 @@ BuildArch:  noarch
 %obsolete_ticket https://github.com/terrapkg/packages/issues/991
 %obsolete iosevka-fusion-fonts 25.1.1-2
 
+%obsolete_ticket https://github.com/terrapkg/packages/pull/2522
+%obsolete terra-gnome-shell-extension-pop-shell 1.2.0^20241010.e25621e-2
```
Comment (madonuko) [inline anda/terra/obsolete/terra-obsolete.spec]: EVR (epoch, version, release) needs to be strictly higher than the last EVR of the package

Comment (sadlerm4) [reply anda/terra/obsolete/terra-obsolete.spec:125]: That defeats the purpose though, I specifically want to obsolete the current release of the package, not future versions, which won't get built anyway when the spec is removed?

**Comment by sadlerm4** [comment]
Actually, on second thoughts, can you just remove the rpms from the server? I forgot that the 2nd release of the latest version is also uninstallable on 41.

Since it can be guaranteed that the package is not installed on any system running 41, do I even still need to obsolete it?

**Comment by madonuko** [comment]
@lleyton see above, consider satm rm from 41, rawhide: terra-gnome-shell-extension-pop-shell-0:1.2.0^20241010.e25621e-2.fc41.noarch.rpm

**Comment by madonuko** [comment]
@sadlerm4 I am not so sure but I think we need to obsolete this on only frawhide.

**Comment by sadlerm4** [comment]
i didn't realise rawhide is an actual repo too, so what i'm asking is to remove the rpms from terrarawhide and terra41

alternatively i can just fix the spec so it's installable lol

and then revisit this with lea some other time (y)

to clarify the reason it can't be installed on 41+ is because the gnome-shell version required is unsatisfiable because i uh messed it up (shhh)

**Comment by madonuko** [comment]
yeah lea removed this from rawhide and 41 already so

**Comment by sadlerm4** [comment]
okay so then i don't need to do anything right? just need to get this PR approved and then we're good:)

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2591)|
|❌|el10|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
backport --pr 2522
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/12254555375) for details

**Pull request merged**